src/Makefile.legacy

```diff
@@ -1759,10 +1757,6 @@ $(SUBDIRS):
 ../run/unique.com: john.com
 	copy john.com ..\run\unique.com
 
-# This must not be specified with path because "make clean" would delete it.
-john.local.conf:
-	[ -f ../run/john.local.conf ] || touch ../run/john.local.conf
-
```

Comment:
BTW: Autoconf isn't just creating an *empty* run/kernels directory for non-OpenCL-builds!
Instead, it contains 96 files (*.h and *.cl).
This is somewhat unexpected.